### PR TITLE
feat: added logs quick filters product docs

### PR DESCRIPTION
**Pull request by vikrantgupta25**
- added docs for logs quick filters ui

### CODE REVIEW

**Comment by vikrantgupta25** [comment]
contributes to - https://github.com/SigNoz/engineering-pod/issues/1946

File: constants/docsSideNav.ts [hunk 1]
```diff
@@ -670,6 +670,11 @@ const docsSideNav = [
             route: '/docs/userguide/logs_query_builder',
             label: 'Logs Query Builder',
           },
+          {
+            type: 'doc',
+            route: '/docs/userguide/logs_quick_filters',
```
Comment (Calm-Rock) [inline constants/docsSideNav.ts]:
```suggestion
            route: '/docs/userguide/logs-quick-filters',
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+date: 2024-12-02
+title: Logs Quick Filters
+id: logs_quick_filters
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
id: logs-quick-filters
```

Please change the file name to `logs-quick-filters.mdx` as well.

File: data/docs/userguide/logs_quick_filters.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]: Can you please put this file inside logs-management folder under a sub folder called `features`

**Review by nityanandagohain — COMMENTED** [review]
Also add a paragraph that if you want to map these attributes from logs body or another attribute then use logs pipelines and a reference for this https://signoz-web-git-logs-quick-filter-docs-signoz.vercel.app/docs/logs-pipelines/introduction/

File: data/docs/userguide/logs_quick_filters.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+date: 2024-12-02
+title: Logs Quick Filters
+id: logs_quick_filters
+---
+
+The Logs Explorer comes with quick filters which allows filtering of logs based on key attributes making it easier to analyze and debug applications.
+Here's a detailed explaination of the available filters and how to use them -
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
Below is a detailed explanation of the available filters and instructions on how to use them.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  <img
+    src="/img/docs/logs-management/ui/logs-quick-filters.webp"
+    alt="Quick Filters in Logs Explorer"
+  />
+  <figcaption>
+    <i> Quick Filters in Logs Explorer </i>
+  </figcaption>
+</figure>
+
+### Available Filters
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
## Available Filters
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+1. **Severity Text**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
**Severity Text**:
```
```suggestion
### Severity Text
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+   - **Description**: Filter logs based on their severity text.
+   - **Usage**: Useful for quickly filtering in FATAL logs when some incident occurs or filter in DEBUG logs when debugging some issue.
+   - **OTEL Attribute**: Maps to `severity_text` opentelemetry attribute.
+2. **Environemnt**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### Environment
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+   - **Description**: Filter logs based on the source environment.
+   - **Usage**: Useful for filtering in/out logs coming from different environments. Example - Production / Staging / Development etc.
+   - **OTEL Attribute**: Maps to `deployment.environment` opentelemetry resource attribute.
+
+3. **Service Name**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### Service Name
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+   - **Description**: Filter logs by the name of the service that generated them.
+   - **Usage**: Select from a list of service names to isolate logs from a specific service. This can be helpful in microservices architectures where multiple services generate logs.
+   - **OTEL Attribute**: Maps to `service.name` opentelemetry resource attribute.
+
+4. **Hostname**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### Hostname
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+   - **Description**: Filter logs based on the hostname of the machine that generated the logs.
+   - **Usage**: This filter allows you to isolate logs from specific hosts, which is particularly useful in environments with multiple servers or instances. For example, you can filter logs to view only those generated by a particular server during a specific time frame.
+   - **OTEL Attribute**: Maps to `host.name` opentelemetry resource attribute.
+
+5. **K8s Pod Name**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### K8s Pod Name
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+   - **Description**: Filter logs based on the name of the Kubernetes pod that generated them.
+   - **Usage**: This filter allows you to isolate logs from specific pods, which is particularly useful in environments with multiple pods running the same service.
+   - **OTEL Attribute**: Maps to `k8s.pod.name` opentelemetry resource attribute.
+
+6. **K8s Node Name**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### K8s Node Name
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+
+   - **Description**: Filter logs based on the name of the Kubernetes node that generated them.
+   - **Usage**: This filter helps in isolating logs from specific nodes, which can be useful for debugging issues related to particular nodes in a cluster.
+   - **OTEL Attribute**: Maps to `k8s.node.name` opentelemetry resource attribute.
+
+7. **K8s Cluster Name**:
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
### K8s Cluster Name
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+   - **Description**: Filter logs based on the name of the Kubernetes cluster that generated them.
+   - **Usage**: This filter allows you to isolate logs from specific clusters, which is particularly useful in environments with multiple clusters running different services.
+   - **OTEL Attribute**: Maps to `k8s.cluster.name` opentelemetry resource attribute.
+
+<Admonition>
+  For mapping attributes from log body to the above attributes then use{' '}
+  <a href="https://signoz.io/docs/logs-pipelines/introduction/">log pipelines</a>
+</Admonition>
+
+### How to use quick filters
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
## How to use quick filters
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+   - **Usage**: This filter helps in isolating logs from specific nodes, which can be useful for debugging issues related to particular nodes in a cluster.
+   - **OTEL Attribute**: Maps to `k8s.node.name` opentelemetry resource attribute.
+
+7. **K8s Cluster Name**:
+   - **Description**: Filter logs based on the name of the Kubernetes cluster that generated them.
+   - **Usage**: This filter allows you to isolate logs from specific clusters, which is particularly useful in environments with multiple clusters running different services.
+   - **OTEL Attribute**: Maps to `k8s.cluster.name` opentelemetry resource attribute.
+
+<Admonition>
+  For mapping attributes from log body to the above attributes then use{' '}
```
Comment (nityanandagohain) [inline data/docs/userguide/logs_quick_filters.mdx]: For mapping attributes from log body to the above attributes ~~then~~ use

remove `then`

File: data/docs/userguide/logs_quick_filters.mdx [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+    <i> Quick Filters in Logs Explorer </i>
+  </figcaption>
+</figure>
+
+### Available Filters
+
+1. **Severity Text**:
+   - **Description**: Filter logs based on their severity text.
+   - **Usage**: Useful for quickly filtering in FATAL logs when some incident occurs or filter in DEBUG logs when debugging some issue.
+   - **OTEL Attribute**: Maps to `severity_text` opentelemetry attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`severity_text`](https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext) opentelemetry attribute.
```
https://opentelemetry.io/docs/contributing/style-guide/#opentelemetryio-word-list

Please confirm if this hyperlink is correct - https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext

File: data/docs/userguide/logs_quick_filters.mdx [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+2. **Environemnt**:
+
+   - **Description**: Filter logs based on the source environment.
+   - **Usage**: Useful for filtering in/out logs coming from different environments. Example - Production / Staging / Development etc.
+   - **OTEL Attribute**: Maps to `deployment.environment` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`deployment.environment`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/deployment/#deployment-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+
+3. **Service Name**:
+
+   - **Description**: Filter logs by the name of the service that generated them.
+   - **Usage**: Select from a list of service names to isolate logs from a specific service. This can be helpful in microservices architectures where multiple services generate logs.
+   - **OTEL Attribute**: Maps to `service.name` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`service.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/service/#service-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+
+4. **Hostname**:
+
+   - **Description**: Filter logs based on the hostname of the machine that generated the logs.
+   - **Usage**: This filter allows you to isolate logs from specific hosts, which is particularly useful in environments with multiple servers or instances. For example, you can filter logs to view only those generated by a particular server during a specific time frame.
+   - **OTEL Attribute**: Maps to `host.name` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`host.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host/#host-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+
+5. **K8s Pod Name**:
+
+   - **Description**: Filter logs based on the name of the Kubernetes pod that generated them.
+   - **Usage**: This filter allows you to isolate logs from specific pods, which is particularly useful in environments with multiple pods running the same service.
+   - **OTEL Attribute**: Maps to `k8s.pod.name` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`k8s.pod.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/k8s/#kubernetes-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+
+6. **K8s Node Name**:
+
+   - **Description**: Filter logs based on the name of the Kubernetes node that generated them.
+   - **Usage**: This filter helps in isolating logs from specific nodes, which can be useful for debugging issues related to particular nodes in a cluster.
+   - **OTEL Attribute**: Maps to `k8s.node.name` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`k8s.node.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/k8s/#kubernetes-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+
+7. **K8s Cluster Name**:
+   - **Description**: Filter logs based on the name of the Kubernetes cluster that generated them.
+   - **Usage**: This filter allows you to isolate logs from specific clusters, which is particularly useful in environments with multiple clusters running different services.
+   - **OTEL Attribute**: Maps to `k8s.cluster.name` opentelemetry resource attribute.
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]:
```suggestion
   - **OTel Attribute**: Maps to [`k8s.cluster.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/k8s/#kubernetes-attributes) resource attribute.
```

File: data/docs/userguide/logs_quick_filters.mdx [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+    src="/img/docs/logs-management/ui/logs-quick-filters.webp"
+    alt="Quick Filters in Logs Explorer"
+  />
+  <figcaption>
+    <i> Quick Filters in Logs Explorer </i>
+  </figcaption>
+</figure>
+
+### Available Filters
+
```
Comment (Calm-Rock) [inline data/docs/userguide/logs_quick_filters.mdx]: There are multiple filtering options available, with each filter mapped to a specific [OpenTelemetry (OTEL) attribute](https://opentelemetry.io/docs/specs/otel/common/#attribute). Here’s a list of the available filters:

File: data/docs/userguide/logs_quick_filters.mdx (hunk 14 above)
Comment (vikrantgupta25) [reply data/docs/userguide/logs_quick_filters.mdx]: done @nityanandagohain

**Review by Calm-Rock — CHANGES_REQUESTED** [review]

**Comment by vikrantgupta25** [comment]
TODO[AI] - check the `deployment.environment` attribute as this has been deprecated. cc @Calm-Rock

**Review by Calm-Rock — APPROVED** [review]

**Pull request merged**